### DISCUSSION TITLE
There is a problem with the rspamd service.

### DISCUSSION THREAD

**Discussion by Toni202012569** [postalserver/postal#2660] [Help with using Postal]
Good afternoon. I have a problem with the rspamd service. When I create a server and specify domains in it, the rspamd program receives information and scans letters. However, if I create domain records at the organization level, then rspamd no longer works and does not check any mail.
![Скриншот 19-10-2023 10 50 01](https://github.com/postalserver/postal/assets/139015871/5766661c-e7d6-434a-a06b-7e495769ac36)

**Comment by willpower232** [comment]
Have you definitely set this setting under Advanced Settings?

![image](https://github.com/postalserver/postal/assets/1619102/318e7b9d-a7d2-4899-a33b-adf544d10bab)

**Reply by Toni202012569** [reply]
Good evening. No, I did not fill out anything in this field. Thanks for the information, I'll try to indicate now. On my first server the number is 12.0, is this normal or do I need to other numbers else?

**Reply by willpower232** [reply]
its hard to say what number you should use, ordinarily you can trust outgoing email and 12 sounds quite high but you'd have to have a look at the actual emails going through your Postal to see what scores they got and decide for yourself what limit you want to set
